server/gitrest/packages/gitrest-base/src/utils/helpers.ts

```diff
@@ -180,3 +193,12 @@ export function logAndThrowApiError(error: any, request: Request, params: IRepoM
     // is actively throwing NetworkErrors with appropriate status codes according to what the protocols expect.
     throw new NetworkError(400, `Error when processing ${request.method} request to ${request.url}`);
 }
+
+export async function getRepoManagerFroWriteAPI(
```

Comment:
>Fro [](http://example.com/codeflow?start=36&length=3)

What is Fro? :)

Reply:
Something that should have not been here 😄  Thanks for point out, it was a typo